projects/lwan/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+
+mkdir -p $WORK/lwan
+cd $WORK/lwan
+
+cmake -GNinja \
+	-DCMAKE_BUILD_TYPE=Debug -DSANITIZER=${SANITIZER} \
+	-DCMAKE_C_COMPILER="${CC}" -DCMAKE_C_FLAGS="${CFLAGS}" \
+	$SRC/lwan/
+
+ninja -v liblwan.a
+
+zip -jr $OUT/request_fuzzer_seed_corpus.zip $SRC/lwan/fuzz/corpus/corpus-request-*
+
+for fuzzer in $SRC/lwan/src/bin/fuzz/*_fuzzer.cc; do
+	executable=$(basename $fuzzer .cc)
+	$CXX $CXXFLAGS -std=c++11 \
+		-Wl,-whole-archive $WORK/lwan/src/lib/liblwan.a -Wl,-no-whole-archive \
+		-I$SRC/lwan/src/lib $fuzzer \
+		-fsanitize=${SANITIZER} \
```

Comment:
This is not needed, the necessary sanitizer flags are in `$CXXFLAGS` already

Reply:
Ack.